### DISCUSSION TITLE
ComboBox

### DISCUSSION THREAD

**Discussion by zSynctic** [TomSchimansky/CustomTkinter#1645] [Q&A] [answered]
I have a problem with the ComboBox its probably something I don't know but here it is:

I want to make it that even if the value is not written in the values list it can be used if its just written

**Comment by Akascape** [comment, marked as answer]
@zSynctic 
> I want to make it that even if the value is not written in the values list it can be used if its just written

bind the combobox with enter button.
```python
combobox.bind("<Return>", lambda e: command())
...
```
So, now the user can use the custom option after pressing the `Enter` key,

**Reply by zSynctic** [reply]
I'm sorry for replying late but how would I use that to make a command to get what is written in the combo box and set it into the combobox variable this is the first time I'm using the combobox

**Reply by Akascape** [reply]
> how would I use that to make a command to get what is written in the combo box and set it into the combobox variable

@zSynctic 
You can get the written text using `combobox.get()` method. 

If you want to append that text inside the combobox options, then use this:
```python
value_list = [ ] # the list of options for combobox
def command()
     if combobox.get() in value_list: 
          return # don't proceed if the value exist
     value_list.append(combobox.get())
     combobox.configure(values=value_list)
```
This command function will be called when you write something and press `enter` key in combobox.
